src/doc/gui_manual/Quantitative/Lineout.rst

```diff
@@ -92,6 +91,260 @@ off the **Apply operators to all plots** check box in the **Main Window**
+
+All the images associated with the examples can be generated with the
+script lineout.py.
+
+Zonal centering with sampling
+"""""""""""""""""""""""""""""
+
+In the case of sampling, the step function will become more and more apparent
+as the number of sample points increases.
+
+In the example below there are only 12 samples points and the step function
+is only somewhat apparent, since the number of sample points within a cell
+ranges between one and three.
+
+.. _lineoutexample2:
+
+.. figure:: images/rect2d_d_12_sampled.png
+
+   A zonal variable with relatively few sample points.
+
+In the example below there are 60 sample points and the step function is
+quite apparent.
+
+.. _lineoutexample3:
+
+.. figure:: images/rect2d_d_60_sampled.png
+
+   A zonal variable with a large number of sample points.
+
+Zonal centering without sampling
+""""""""""""""""""""""""""""""""
+
+In the case of non-sampling, the sample points are chosen where the line
+intersects cell boundaries, which are lines in 2D and faces in 3D. The first
+point of the line has the zonal value of the cell it is within and the
+remaining points have the value of the cell the line is about to enter.
+In this case the step function nature of the variable is completely lost.
+
+In the example below the sample points are placed based on where the line
+intersects the edges of the cells. The step function nature of the variable
+is completely lost and the line looks smoother than the sampled case.
+
+.. _lineoutexample4:
+
+.. figure:: images/rect2d_d_nonsampled.png
+
+   A zonal variable without sampling.
+
+Nodal variables vary linearly within a cell. Using sampling produces high
+quality results as long as the number of sample points is chosen such
+that all the cells along the line contain at least one sample point. Using
+non sampling tends to produce poor results based on its interpolation
+method (described below) and may result in jagged lines, even for smoothly
+varying functions.
+
+Nodal centering with sampling
+"""""""""""""""""""""""""""""
+
+In the example below the 12 samples points do a good job of capturing
+the data along the line since all the cells are sampled at least once.
+
+.. _lineoutexample5:
+
+.. figure:: images/rect2d_d2_12_sampled.png
+
+   A nodal variable with relatively few sample points.
+
+Increasing the number of sample points in this case doesn't change the
+shape of the curve.
+
+.. _lineoutexample6:
+
+.. figure:: images/rect2d_d2_60_sampled.png
+
+   A nodal variable with many sample points.
+
+Nodal centering without sampling
+""""""""""""""""""""""""""""""""
+
+In the example below the sample points are placed based on where the line
+intersects the edges of the cells. The first point of the line has the
+average of the nodes of the cell that the point is within and the remaining
+points have the value of the average of the nodes of the cell the line is
+about to enter. This can lead to a jadded line even for a smoothly varying
```

Comment:
jagged, not jadded.
Sorry I missed it my previous look.

Reply:
Corrected.